### ISSUE TITLE
Wrong sigma for functions

### ISSUE THREAD

**Issue by b3liever** [peterroelants/peterroelants.github.io#17]
https://github.com/peterroelants/peterroelants.github.io/blob/6ffce2cf3927f0d20ef763fd6939322531c06188/_posts/2015-06-10-cross-entropy-softmax.html#L105

Greek letter ``ς`` is only used as last letter in words that end with an s sound (weird rule I know). In math
to denote functions we use regular sigma ``σ`` (in mathml \sigma).
